rest/src/test/java/com/carhouse/rest/JsonConverter.java

```diff
@@ -0,0 +1,20 @@
+package com.carhouse.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonConverter {
```

Comment:
Are you sure that you need to have this class?
Think only `ObjectMapper` is enough in context